assets/admin/editor-wizard/wizard.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Wizard component.
+ *
+ * @param {Object}   props
+ * @param {Array}    props.steps        Array with the steps that will be rendered.
+ * @param {Function} props.onCompletion Callback to call when wizard is completed.
+ */
+const Wizard = ( { steps, onCompletion } ) => {
+	const [ currentStepNumber, setCurrentStepNumber ] = useState( 0 );
+	const [ data, setData ] = useState( {} );
+
+	const CurrentStep = steps[ currentStepNumber ];
+
+	const goToNextStep = () => {
+		if ( currentStepNumber + 1 < steps.length ) {
+			setCurrentStepNumber( currentStepNumber + 1 );
+		} else {
+			onCompletion( data );
+		}
+	};
+
+	return (
+		<div className={ 'sensei-editor-wizard' }>
+			<div className={ 'sensei-editor-wizard__body' }>
+				<CurrentStep
+					data={ data }
+					setData={ setData }
+					onCompletion={ onCompletion }
+				/>
+			</div>
+			<div className={ 'sensei-editor-wizard__footer' }>
+				<div className={ 'sensei-editor-wizard__progress' }>
+					Step { currentStepNumber } of { steps.length }
```

Comment:
I think we should display `currentStepNumber + 1`, right?

Reply:
Correct. Changed here: 119ea9691

Thanks!